docs/src/modules/ROOT/pages/integration/config-solver-properties.adoc

```diff
@@ -0,0 +1,52 @@
+////
+Quarkus and Spring Boot support the same configuration properties.
```

Comment:
Good idea.

One thing that's not entirely obvious to me is the `<solverName>` part. You explain it in another section, and that is fine. But do you ever mention that this is optional? That `<solverName>` can be exluded if you only ever inject one thing?

I think this needs to be described in more detail, to give more clarity.